### DISCUSSION TITLE
Issue with lasergrbl program

### DISCUSSION THREAD

**Discussion by guyslc60A** [arkypita/LaserGRBL#1821] [Q&A]
Can anyone tell me why my design is in the far-right corner?
![image](https://user-images.githubusercontent.com/105014985/167008492-27b0044e-66a2-4349-ac95-9d4482e66e60.png)

**Comment by Tribo-2dot0** [comment]
Open it in Inkscape, select the image, Edit > Fit Page to Selection.
File > Export Bitmap Image...
